index.html

```diff
@@ -14,19 +27,23 @@
   </head>
   <body>
     <header>
-      <nav>
-        <a href="index.html">
-          <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
-        </a>
-        <a class="cta cta-short" href="signin.html">
-          <span>로그인</span>
-        </a>
-      </nav>
+      <div class="nav-container">
+        <nav>
```

Comment:
```suggestion
        <nav class="nav-container">
```

이렇게 하셔도 될거같아요 추가로, 스타일을 위해 구조를 바꾸는 것 == nav-container스타일을 위해, div로 감싸는 것보다, 작은 변화로 적용할 수 있으니까요